charts/kyverno/README.md

```diff
@@ -67,7 +67,10 @@ The following table lists the configurable parameters of the kyverno chart and t
 
 | Parameter                          | Description                                                                                                                                                                                                                                              | Default                                                                                                                                                                                                                                                                  |
 | ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `antiAffinity`                     | node/pod affinities. Enabled by default but can be disabled in single-node clusters.                                                                                                                                                                                                                                      | `nil`                                                                                                                                                                                                                                                                    |
+| `antiAffinity.enable`              | pod antiAffinities toggle. Enabled by default but can be disabled if you want to schedule pods to the same node                                                                                                                                          | `true`                                                                                                                                                                                                                                                                   |
+| `podAntiAffinity`                  | pod antiAffinities default values. can be overwrite                                                                                                                                                                                                      | `Pod Anti Affinity`                                                                                                                                                                                                                                                      |
```

Comment:
Minor nitpick, but the parameters in this list are ordered alphabetically.  So be good to move the `podAntiAffinity`, `podAffinity` and `nodeAffinity` to the same place.

Reply:
You are right. Done!